File: fizzbuzz/jasmine/spec/fizzBuzzSpec.js
describe("fizzBuzz", function() {

  it("should print fizz for multiples of 3", function() {
    expect(fizzBuzz(1, 3)).toEqual([1, 2, "fizz"]);
  });

  it("should print buzz for multiples of 5", function() {
    expect(fizzBuzz(1, 5)).toEqual([1, 2, "fizz", 4, "buzz"]);
  });

  it("should print fizzbuzz for multiples of both 3 and 5", function() {
    expect(fizzBuzz(12, 16)).toEqual(["fizz", 13, 14, "fizzbuzz", 16]);
  });

});

File: fizzbuzz_dir/spec/fizzbuzz_spec.rb
require './fizzbuzz'

describe 'fizzbuzz' do
  it 'returns a number' do
    expect(fizzbuzz(1)).to eq 1
  end
  it 'returns fizz for 3' do
    expect(fizzbuzz(3)).to eq 'fizz'
  end
  it 'returns buzz for 5' do
    expect(fizzbuzz(5)).to eq 'buzz'
  end
  it 'returns fizzbuzz for 15' do
    expect(fizzbuzz(15)).to eq 'fizzbuzz'
  end
end
